packages/githublight/README.md

```diff
@@ -0,0 +1,48 @@
+# GitHub Light
+
+A theme heavily inspired by the [`GitHub light default theme` for VSCode](https://github.com/primer/github-vscode-theme).
+
+## Usage ✨
+
+To enable the theme from the JupyterLab menu bar. Go to `Settings -> JupyterLab Theme -> Github Light`
+
+## Colors
+
+All the colors used for text in the theme are conformant with [WCAG 2.1 contrast standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).
+
+Background color: ![##ffffff](https://via.placeholder.com/20/#ffffff/#ffffff.png) `#ffffff`
+
+Highlight color: ![#321f7ad9](https://via.placeholder.com/20/321f7ad9/321f7ad9.png) `#321f7ad9`
+
+**WCAG compliance**
+
+All the colors in the table shown were tested with the background color.
+
+| Color                                                        | Hex       | Ratio    | Normal text | Large text |
+| ------------------------------------------------------------ | --------- | -------- | ----------- | ---------- |
+| ![#1F2328](https://via.placeholder.com/20/1F2328/1F2328.png) | `#1F2328` | 15.8 : 1 | AAA         | AAA        |
+| ![#6e7781](https://via.placeholder.com/20/6e7781/6e7781.png) | `#6e7781` | 4.5 : 1  | AA          | AAA        |
+| ![#57606a](https://via.placeholder.com/20/57606a/57606a.png) | `#57606a` | 6.4 : 1  | AA          | AAA        |
+| ![#424a53](https://via.placeholder.com/20/424a53/424a53.png) | `#424a53` | 9.0 : 1  | AAA         | AAA        |
+| ![#32383f](https://via.placeholder.com/20/32383f/32383f.png) | `#32383f` | 11.8 : 1 | AAA         | AAA        |
+| ![#24292f](https://via.placeholder.com/20/24292f/24292f.png) | `#24292f` | 14.7 : 1 | AAA         | AAA        |
+| ![#0550ae](https://via.placeholder.com/20/0550ae/0550ae.png) | `#0550ae` | 7.6 : 1  | AAA         | AAA        |
+| ![#0a3069](https://via.placeholder.com/20/0a3069/0a3069.png) | `#0a3069` | 12.8 : 1 | AAA         | AAA        |
+| ![#116329](https://via.placeholder.com/20/116329/116329.png) | `#116329` | 7.4 : 1  | AAA         | AAA        |
+| ![#953800](https://via.placeholder.com/20/953800/953800.png) | `#953800` | 7.4 : 1  | AAA         | AAA        |
+| ![#a40e26](https://via.placeholder.com/20/a40e26/a40e26.png) | `#a40e26` | 7.9 : 1  | AAA         | AAA        |
+| ![#6639ba](https://via.placeholder.com/20/6639ba/6639ba.png) | `#6639ba` | 7.3 : 1  | AAA         | AAA        |
+
+## Font
+
+This theme is using the [Atkinson Hyperlegible font](https://fonts.google.com/specimen/Atkinson+Hyperlegible), which was developed specifically to increase legibility for readers with low vision.
```

Comment:
Some observations:

1. If you are planning to use this typeface for all the themes then you also need to update it in pitaya smoothie to unify the behaviour
2. We should be linking to the Atkinson Hyperlegible website and not google fonts https://brailleinstitute.org/freefont
3. Also we should use the description the typeface uses itself - AFAIK the typeface was designed for readability rather than being better suited for low-vision folks.